meetings/2023-05-10.md

```diff
@@ -0,0 +1,124 @@
+* SC: Spec needs to refer to it before publishing. ED should be sufficient for now. We can nudge W3C Team to follow-up right after merging of 174 to publish the JSON-LD Context.
+
+
+### Solid Protocol Release 0.11.0
+URL: https://github.com/solid/specification/milestone/7
+
+* SC: Have started to mark items towards this milestone.
+* SC: Suggestions and commitments welcome.
+* SC: There are other milestones like FPWD, e.g., https://github.com/solid/specification/milestone/1 , which  I created a long time ago. These are good candidates towards 0.11.0. See also https://github.com/solid/specification/milestone/2 and https://github.com/solid/specification/milestone/3 ..
+* SC: Apparently also moving items from https://github.com/solid/specification/milestone/6 to https://github.com/solid/specification/milestone/7 .
+* SC: When I created these milestones, the idea was to take up the issues based on importance of publishing a REC in the WG. Those maturity levels don't have standing in the CG. If the WG happens it will need to do the same exercise since the maturity levels apply to the WG.
+  
+
+### Solid QA
+URL: https://solidproject.org/ED/qa
+
+* SC: We need to advance Solid QA, including all the related work (e.g., test suite development, tests). It is at the heart of TRs - "gives Solid projects confidence that their software is compatible with other implementations." So, please contribute to the development. Advancing TRs and QA work go together. This is the required ground work in order to inform about the level of adoption of TRs.
+* SC: There is no real spec until we have test suits and implementation reports. If there were no implementation for a feature there is no reason for it to stay in the spec.
+* SC: We have monthly Test Suite panel meeting, next week on Tuesday will be the May meeting. Spec authors and editors need to be closely involved in the test suite work to verify its accuracy. If we are not able to do it here I don't know how are we going to do it in WG.
+
+
+### Update background to clarify Solid and comparisons
+URL: https://github.com/solid/solid-wg-charter/pull/30
+
+* SC: Resolves https://github.com/solid/solid-wg-charter/issues/5
+* SC: There are three thumbs-ups and 1 review so far.
+* SC: Feedback?
+* SC: Any objections to merge?
+* SC: Assigned to PA.
+* eP: I saw PA announced that it is announced to Members, so which version is it - one changing.: https://lists.w3.org/Archives/Public/public-new-work/2023May/0002.html
+* SC: He refers to the advanced notice. It needs to go through the horizontal review and it will come back. For example, Web Accessibility, I18n, TAG may provide some feedback and request clarifications. Once we do that the W3C members will get to vote on the charter, which can result in more issues and PRs. The charter is always in edit mode until it is accepted. Only then W3C will create as snapshot on w3c.org, and even then there can be changes done by re-chartering.
```

Comment:
```suggestion
* eP: I saw PA announced that it is announced to Members, so which version is it — one changing: https://lists.w3.org/Archives/Public/public-new-work/2023May/0002.html
* SC: He refers to the advance notice. It needs to go through horizontal review and it will come back. For example, Web Accessibility, I18n, TAG may provide some feedback and request clarifications. Once we do that, the W3C members will get to vote on the charter, which can result in more issues and PRs. The charter is always in edit mode until it is accepted. Only then will W3C create a snapshot on w3c.org, and even then there can be changes done by re-chartering.
```